### PR TITLE
how-to: add binlog slave client and reparo into 

### PR DESCRIPTION
**Pull request by ericsyh**
### What is changed, added or deleted? 

In current TiDB Binlog doc, we just provide the links of Binlog-Slave-Client and Reparo in TiDB Overview while no link in TOC.But actually these two docs are quite important for users when syncing to Kafka and local file. So i put their links in TOC with TiDB Binlog deployment. 
Also i move the hardware requirements from TiDB Binlog Overview to TiDB Binlog Deployment doc.

### What is the related PR or file link(s)? 

N/A

### Which version does your change affect? 

dev, v3.0, v2.1

### CODE REVIEW

File: v3.0/how-to/deploy/tidb-binlog.md [hunk 1]
```diff
@@ -6,6 +6,15 @@ aliases: ['/docs-cn/tools/binlog/deploy/','/docs-cn/dev/reference/tools/tidb-bin
...
 # TiDB Binlog 集群部署
 
+## 服务器要求
+
+Pump 和 Drainer 都支持部署和运行在 Intel x86-64 架构的 64 位通用硬件服务器平台上。对于开发，测试以及生产环境的服务器硬件配置有以下要求和建议：
+
+| 服务     | 部署数量       | CPU   | 磁盘          | 内存   |
+| :-------- | :-------- | :--------| :--------------- | :------ |
+| Pump | 3 | 8核+   | SSD, 200 GB+ | 16G |
+| Drainer | 1 | 8核+ | SAS, 100 GB+ （如果输出为本地文件，则使用 SSD，并增加磁盘大小） | 16G |
```
Comment (july2993) [inline v3.0/how-to/deploy/tidb-binlog.md]: s/SAS, 100 GB+ （如果输出为本地文件，则使用 SSD，并增加磁盘大小）/如果输出为本地文件:  SAS, 大小视保留数据天数与大小而定/

File: v2.1/how-to/deploy/tidb-binlog.md [hunk 2]
```diff
@@ -5,6 +5,15 @@ category: reference
...
 # TiDB Binlog 集群部署
 
+## 服务器要求
+
+Pump 和 Drainer 都支持部署和运行在 Intel x86-64 架构的 64 位通用硬件服务器平台上。对于开发，测试以及生产环境的服务器硬件配置有以下要求和建议：
+
+| 服务     | 部署数量       | CPU   | 磁盘          | 内存   |
+| :-------- | :-------- | :--------| :--------------- | :------ |
+| Pump | 3 | 8核+   | SSD, 200 GB+ | 16G |
+| Drainer | 1 | 8核+ | SAS, 100 GB+ （如果输出为本地文件，则使用 SSD，并增加磁盘大小） | 16G |
```
Comment (july2993) [inline v2.1/how-to/deploy/tidb-binlog.md]: s/SAS, 100 GB+ （如果输出为本地文件，则使用 SSD，并增加磁盘大小）/如果输出为本地文件:  SAS, 大小视保留数据天数与大小而定/

File: dev/how-to/deploy/tidb-binlog.md [hunk 3]
```diff
@@ -5,6 +5,15 @@ category: reference
...
 # TiDB Binlog 集群部署
 
+## 服务器要求
+
+Pump 和 Drainer 都支持部署和运行在 Intel x86-64 架构的 64 位通用硬件服务器平台上。对于开发，测试以及生产环境的服务器硬件配置有以下要求和建议：
+
+| 服务     | 部署数量       | CPU   | 磁盘          | 内存   |
+| :-------- | :-------- | :--------| :--------------- | :------ |
+| Pump | 3 | 8核+   | SSD, 200 GB+ | 16G |
+| Drainer | 1 | 8核+ | SAS, 100 GB+ （如果输出为本地文件，则使用 SSD，并增加磁盘大小） | 16G |
```
Comment (july2993) [inline dev/how-to/deploy/tidb-binlog.md]: s/SAS, 100 GB+ （如果输出为本地文件，则使用 SSD，并增加磁盘大小）/如果输出为本地文件:  SAS, 大小视保留数据天数与大小而定/

File: v3.0/how-to/deploy/tidb-binlog.md (hunk 1 above)
Comment (ericsyh) [reply v3.0/how-to/deploy/tidb-binlog.md]: Updated in https://github.com/pingcap/docs-cn/pull/1773/commits/b50f76f2867df92fb4ad49354814175446b5ef37

File: v2.1/how-to/deploy/tidb-binlog.md (hunk 2 above)
Comment (ericsyh) [reply v2.1/how-to/deploy/tidb-binlog.md]: Update in https://github.com/pingcap/docs-cn/pull/1773/commits/b50f76f2867df92fb4ad49354814175446b5ef37

File: dev/how-to/deploy/tidb-binlog.md (hunk 3 above)
Comment (ericsyh) [reply dev/how-to/deploy/tidb-binlog.md]: Update in https://github.com/pingcap/docs-cn/pull/1773/commits/b50f76f2867df92fb4ad49354814175446b5ef37

**Comment by lilin90** [comment]
@TomShawn PTAL

File: dev/how-to/deploy/tidb-binlog.md [hunk 4]
```diff
@@ -5,6 +5,15 @@ category: reference
 
 # TiDB Binlog 集群部署
 
+## 服务器要求
+
+Pump 和 Drainer 都支持部署和运行在 Intel x86-64 架构的 64 位通用硬件服务器平台上。对于开发，测试以及生产环境的服务器硬件配置有以下要求和建议：
```
Comment (TomShawn) [inline dev/how-to/deploy/tidb-binlog.md]:
```suggestion
Pump 和 Drainer 均可部署和运行在 Intel x86-64 架构的 64 位通用硬件服务器平台上。在开发、测试和生产环境下，对服务器硬件配置的要求和建议如下：
```

File: dev/how-to/deploy/tidb-binlog.md [hunk 5]
```diff
@@ -5,6 +5,15 @@ category: reference
...
+
+| 服务     | 部署数量       | CPU   | 磁盘          | 内存   |
+| :-------- | :-------- | :--------| :--------------- | :------ |
+| Pump | 3 | 8核+   | SSD, 200 GB+ | 16G |
+| Drainer | 1 | 8核+ | SAS, 100 GB+ （如果输出为本地文件，磁盘大小视保留数据天数而定） | 16G |
```
Comment (TomShawn) [inline dev/how-to/deploy/tidb-binlog.md]: 什么被 “输出为本地文件” 呢？

Comment (ericsyh) [reply dev/how-to/deploy/tidb-binlog.md]: binlog， 可以以本地文件形式存放起来

**Comment by ericsyh** [comment]
@TomShawn Suggestion applied in v3.0/v2.1 in https://github.com/pingcap/docs-cn/pull/1773/commits/2c4319b444459b9300e57e3850955471d9f1d1bf

File: dev/how-to/deploy/tidb-binlog.md (hunk 5 above)
Comment (ericsyh) [reply dev/how-to/deploy/tidb-binlog.md]: 我更新下

**Pull request merged**